Tanks/Assets/Scripts/Camera/CameraControl.cs

```diff
@@ -9,8 +9,8 @@ public class CameraControl : MonoBehaviour
 
     private new Camera camera = null;                        
     private float zoomSpeed = 0;
-    private Vector3 moveVelocity;
-    private Vector3 desiredPosition;             
+    private Vector3 moveVelocity = new Vector3(0.0f, 0.0f, 0.0f);
+    private Vector3 desiredPosition = new Vector3(0.0f, 0.0f, 0.0f);
```

Comment:
Setting these `= default;` will do the same thing. Not necessary to change, just letting you know